Xamarin.Forms.Controls/GalleryPages/ButtonGallery.cs

```diff
@@ -11,7 +11,7 @@ public class ButtonGallery : ContentPage
 		public ButtonGallery ()
 		{
 			//ShellAppearance.SetNavBarVisible(this, false);
-			Shell.SetSearchHandler(this, new SearchHandler() { SearchBoxVisibility = SearchBoxVisiblity.Collapsable });
+			Shell.SetSearchHandler(this, new SearchHandler() { SearchBoxVisibility = SearchBoxVisibility.Collapsable });
```

Comment:
as long as we're in here....
```suggestion
			Shell.SetSearchHandler(this, new SearchHandler() { SearchBoxVisibility = SearchBoxVisibility.Collapsible });
```